### DISCUSSION TITLE
Magic.css There is an error in the code

### DISCUSSION THREAD

**Discussion by jhossyd3v** [cdnjs/cdnjs#14071] [General] [answered]
This is the code on the CDN, instead of "Retourn", it have to say "Return" according to the documentation, actually in the code in the repo on GitHib it is written correct.
![Captura de pantalla (4)](https://user-images.githubusercontent.com/5392981/129794560-3c41f4e6-1ee1-4d44-8f33-d6d5e778b3a0.png)

![image](https://user-images.githubusercontent.com/5392981/129794737-9f79afc3-0ad6-44ac-bdfa-706bb4d89fde.png)

**Comment by MattIPv4** [comment, marked as answer]
👋 We don't provide support for libraries hosted on the CDN, you will need to contact the maintainers of the libraries for issues with them. We replicate their assets as they are tagged, so if there was a bug in a version they tagged, unfortunately, we will have replicated that and they will need to release a new version with a bugfix.

**Reply by jhossyd3v** [reply]
Ok, thanks
